sys/shell/commands/sc_netif.c

```diff
@@ -758,11 +759,13 @@ static int _netif_add(char *cmd_name, kernel_pid_t dev, int argc, char **argv)
         flags |= GNRC_IPV6_NETIF_ADDR_FLAGS_UNICAST;
     }
 
-    if (gnrc_ipv6_netif_add_addr(dev, &addr, prefix_len, flags) == NULL) {
+    if ((ifaddr = gnrc_ipv6_netif_add_addr(dev, &addr, prefix_len, flags)) == NULL) {
         printf("error: unable to add IPv6 address\n");
         return 1;
     }
 
+    gnrc_ipv6_netif_addr_get(ifaddr)->valid = UINT32_MAX; /* Address shall be valid infinitely */
```

Comment:
RPL distributes both the valid and the preferred life times if configured as a root, so when using gnrc_networking manually without the border router configuration it will make a difference